pinot-core/src/main/java/org/apache/pinot/core/common/RowBasedBlockValueFetcher.java

```diff
@@ -43,6 +44,10 @@ public Object[] getRow(int docId) {
     return row;
   }
 
+  public ImmutableRoaringBitmap getColumnNullBitmap(int colId) {
```

Comment:
If null handling is enabled, we can directly set `null` in `getRow()` instead of relying on the caller to fill the null values